src/EventsOfWeek/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+
+namespace EventsOfWeek
+{
+    class Program
+    {
+        enum Week
+        {
+            Monday = 1,
+            Tuesday = 2,
+            Wednesday = 3,
+            Thursday = 4,
+            Friday = 5,
+            Saturday = 6,
+            Sunday = 7,
+        }
+        static string[] arrayWithEvents = new string[7];
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.WriteLine("Hello, dear friend.");
+                Console.WriteLine("You can select the day by entering its name or number.");
+                Console.WriteLine("Example: 'Monday', 'mon', '1' (Remember that monday - 1, tuesday - 2, etc.).");
+                Console.WriteLine("Good luck!");
+                Console.WriteLine("**************************************************************************");
+                SelectDay();
+                Console.WriteLine("Input 'q' for exit, any other input will restart app");
+                if (Console.ReadLine().Equals("q"))
+                {
+                    return;
+                }
+                Console.Clear();
+            }
+        }
+        static void SelectDay()
+        {
+            string userInput = Console.ReadLine();
+            switch (userInput.ToLower())
+            {
+                case "mon":
+                case "monday":
+                case "1":
+                    Console.WriteLine("U choose monday.");
+                    AddEvent(Convert.ToInt32(Week.Monday));
```

Comment:
Try to send to method enum param. It is look like: AddEvent((int)Week.Monday);
Or try to work with enum at AddEvent method (this case better).